src/utils/helpers.ts

```diff
@@ -155,10 +156,13 @@ export function getLatestDiscussionComment(
     return null;
   }
 
-  return comments
-    .flatMap((comment) => comment.replies.nodes)
-    .concat([comments[comments.length - 1]])
-    .reduce((a, b) => (a.createdAt > b.createdAt ? a : b));
+  // Return latest reply if available
+  if (comments[0].replies.nodes.length === 1) {
+    return comments[0].replies.nodes[0];
```

Comment:
So it already comes in the right order? Why were we doing all of that logic then 😂

Reply:
Exactly!